packages/modals/package.json

```diff
@@ -11,12 +11,15 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "0.2.0",
+  "version": "0.3.0",
   "module": "lib/index.esm.js",
   "main": "lib/index.js",
   "sideEffects": false,
   "files": [
-    "/lib"
+    "lib",
+    "src",
```

Comment:
do we always import src in the npm package?

Reply:
by looking on existing pacakges, yes